registration-processor/pre-processor/registration-processor-packet-validator-stage/src/test/java/io/mosip/registration/processor/stages/utils/XSDValidationTest.java

```diff
@@ -0,0 +1,115 @@
+	@Before
+	public void setUp() throws Exception {
+		
+        ReflectionTestUtils.setField(xsdValidation, "configServerFileStorageURL", "http://localhost:51000/config/registration-processor/mz/master/");
+        ReflectionTestUtils.setField(xsdValidation, "schemaName", "mosip-cbeff.xsd");
+        ClassLoader classLoader = getClass().getClassLoader();
+		File xsdFile = new File(classLoader.getResource("mosip-cbeff.xsd").getFile());
+		InputStream xsd = new FileInputStream(xsdFile);
+		URL u = PowerMockito.mock(URL.class);
+        String url = "http://localhost:51000/config/registration-processor/mz/master/mosip-cbeff.xsd";
+        PowerMockito.whenNew(URL.class).withArguments(url).thenReturn(u);
+        PowerMockito.when(u.openStream()).thenReturn(xsd);
+        List<io.mosip.kernel.biometrics.entities.BIR> birTypeList = new ArrayList<>();
+        io.mosip.kernel.biometrics.entities.BIR birType1 = new BIR.BIRBuilder().build();
+        io.mosip.kernel.biometrics.entities.BDBInfo bdbInfoType1 = new BDBInfo.BDBInfoBuilder().build();
+        io.mosip.kernel.biometrics.entities.RegistryIDType registryIDType = new RegistryIDType("HMAC", "SHA-256");
+        io.mosip.kernel.biometrics.constant.QualityType quality = new QualityType();
+        quality.setAlgorithm(registryIDType);
+        quality.setScore(100l);
+        bdbInfoType1.setQuality(quality);
+        BiometricType singleType1 = BiometricType.FINGER;
+        List<BiometricType> singleTypeList1 = new ArrayList<>();
+        singleTypeList1.add(singleType1);
+        List<String> subtype1 = new ArrayList<>(Arrays.asList("Left", "Thumb"));
+        bdbInfoType1.setSubtype(subtype1);
+        bdbInfoType1.setType(singleTypeList1);
+        bdbInfoType1.setLevel(ProcessedLevelType.RAW);
+        bdbInfoType1.setPurpose(PurposeType.ENROLL);
+        bdbInfoType1.setCreationDate(LocalDateTime.now());
+        RegistryIDType format=new RegistryIDType();
+        format.setOrganization("Mosip");
+        format.setType("7");
+        bdbInfoType1.setFormat(format);
+        birType1.setBdbInfo(bdbInfoType1);
+        birType1.setBdb("bdb".getBytes());
+        VersionType versionType=new VersionType(1, 1);
+        birType1.setVersion(versionType);
+        birType1.setCbeffversion(versionType);
+        BIRInfoBuilder birInfoBuilder=new BIRInfoBuilder();
+        birInfoBuilder.withIntegrity(true);
+        birType1.setBirInfo(new BIRInfo(birInfoBuilder));
+        birTypeList.add(birType1);
+        biometricRecord.setBirInfo(new BIRInfo(birInfoBuilder));
+        biometricRecord.setCbeffversion(versionType);
+        biometricRecord.setVersion(versionType);
+        biometricRecord.setSegments(birTypeList);
+        PowerMockito.mockStatic(CbeffValidator.class);
+        PowerMockito.when(CbeffValidator.createXMLBytes(Mockito.any(), Mockito.any())).thenReturn("bdd".getBytes());
+	}
+	
+	
+	@Test
+	public void TestvalidateXSD() throws IOException, Exception {

```

Comment:
normally unit test methods start with small t rt ?

Reply:
I have changed it.